.gitignore

```diff
@@ -9,3 +9,6 @@ lib
 
 # Dependencies
 node_modules
+
+# Editor files
+*.swp
```

Comment:
Editor-specific files must be ignored by dev's global .gitignore. I don't think we need to update local .gitignore to respect that.

Such files are usually tracked in code review. So it is safe to remove this change (only delete file itself)